### DISCUSSION TITLE
Remove DRM from WOOK epubs

### DISCUSSION THREAD

**Discussion by helzayat** [apprenticeharper/DeDRM_tools#2089] [Q&A]
Can DeDRM be used with epubs from WOOK? I purchased a book from them that I can read in their Android or iOS app or in a browser on windows. The epub file (found in the Android app's storage) is encoded somehow.
Has anyone had any experience with the (Portuguese) platform? Any other tools to allow me to export my book to a Kindle for example?
Thanks in advance
[book.zip](https://github.com/apprenticeharper/DeDRM_tools/files/9035209/book.zip)
.

**Comment by ElleKayEm** [comment]
The only epub DRM this works with is Adobe and Nook (Barnes & Noble).
